style.css

```diff
@@ -486,9 +498,89 @@ img {
   border-radius: 8px;
   margin-top: 10px;
   margin-left: 5px;
+  width: 327px;
+  height: 170px;
 }
 
 #get-in-touch-button {
   margin-top: 12px;
   margin-left: 5px;
 }
+
+@media only screen and (max-width: 768px) {
+  .desktop-toolbar {
+    display: none;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  .mobile-toolbar > nav,
+  .icon {
+    display: none;
+  }
+
+  .desktop-toolbar > nav {
+    font-family: 'Poppins', sans-serif;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 10em;
+    margin: 1em 1em 1em 1em;
+  }
+
+  .desktop-toolbar-secondary {
+    display: flex;
+    flex-direction: row;
+    gap: 30px;
+    font-family: 'Poppins', sans-serif;
+    font-size: 2rem;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 1.4rem;
+    margin: 1.2rem 5rem;
+    align-items: center;
+    align-content: space-between;
+    justify-content: space-between;
+    color: #344563;
+    list-style-type: none;
+  }
+
+  .icon {
+    display: block;
+    margin: auto;
+  }
+
+  .work {
+    display: flex;
+    flex-direction: row;
+  }
+
+  #about-myself {
+    display: flex;
+    flex-flow: row wrap;
+  }
+
```

Comment:
- [ ]  Kindly make sure your about section is centred and looks as close as possible to the Figma design 🙏
- [ ] Please make sure you give the about section a `border-top-right-radius` as seen in the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/186784492-1f795a9e-a093-475e-b786-5e8ee9d8fa49.png)  |  ![image](https://user-images.githubusercontent.com/74814780/186784543-12e27876-fb01-473d-bba3-bf627a70620e.png)

Reply:
Done!